usaspending_api/api_contracts/contracts/v2/disaster/cfda/loans.md

```diff
@@ -67,7 +67,7 @@ Returns loan spending details of CFDA Programs receiving supplemental funding bu
 
 ## Filter (object)
 + `def_codes` (required, array[DEFC], fixed-type)
-+ `award_type_codes` (optional, array[string], fixed-type)
++ `award_type_codes` (optional, array[enum[string]], fixed-type)
     + Only accepts loan award type `07` or `08` in the array, since this endpoint is specific to loans
```

Comment:
These contracts aren't passing Dredd.  It looks like we've done descriptions wrong in a whole mess of places.  Turns out, any descriptive text should not be preceded by a `+`.  The only reason these failed is because they're mixing the text description with integer `07` and `08` values which caused Python to throw a fit.

You can see a description of how enums should work here: https://apiblueprint.org/documentation/specification.html#uri-parameters-section

Even though it's talking about parameters, the syntax is the same.

Reply:
Ok, not as many as I thought.  These three contracts and `disaster/agency/spending.md`.  Would you mind horribly correcting it there as well?

Reply:
ALSO, and this is an issue from before, `sort` is not accepting `id` as a sort order.  I don't know if that's an issue with the contract or the endpoint.  Should it sort on `id`?

`Field 'sort' is outside valid values ['description', 'count', 'obligation', 'outlay', 'face_value_of_loan']`

Reply:
I can go through and clean these up. The `id` field is not sortable due to limitations of us using pre-aggregated "agg_key" to make Elasticsearch more performant so I will remove that.